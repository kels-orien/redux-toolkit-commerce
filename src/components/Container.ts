import styled from 'styled-components'



const Container = styled.div`
    display: flex;
    flex-flow: wrap;
    justify-content: center;
`

export default Container