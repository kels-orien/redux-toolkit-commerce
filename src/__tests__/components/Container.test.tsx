import React from 'react'
import renderer from 'react-test-renderer'
import Container from '../../components/Container'

test('Container render correctly', () => {
    const tree = renderer.create(<Container/>).toJSON();

    expect(tree).toMatchSnapshot()

})