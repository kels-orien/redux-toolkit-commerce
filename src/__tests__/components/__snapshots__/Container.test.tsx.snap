// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container render correctly 1`] = `
<div
  className="sc-bdnxRM dmHvCG"
/>
`;
